careers/technical-engineer.md

```diff
@@ -16,63 +16,64 @@ The senior engineer displays a mastery of the core competencies of software engi
 
 **Collaboration:** They ensure that they do not become a single point of failure for their domain by mentoring and training others. When relevant, they work effectively with others outside the organization (whether in other departments or with vendors). They are capable of effectively running projects of any size independently.
 
-## Staff Engineer (IC 6)
+### Responsibilities
 
-**Technical Direction:** Technology cannot speak for itself and requires effective advocates on its behalf. Staff engineers successfully advance technology by being pragmatic, deliberate, and focused more on the long-term trend of progress than viewing each individual decision as a make-or-break crisis. Staff engineers essentially act like part-time product managers for technology. They are able to get everyone focused on a goal and lay out the steps to get there.
+* Provide technical leadership and guidance across the product and engineering organizations.
+* Make key architectural decisions and ensure the scalability, performance, and reliability of critical systems.
+* Mentor and coach junior and mid-level engineers to foster their technical growth.
+* Collaborate with product managers, designers, and stakeholders to define requirements and solutions.
+* Drive the technical vision and strategy for your area of responsibility, aligning with organizational goals.
+* Conduct thorough code reviews to continually improve the overall quality of our codebase.
+* Take ownership of significant software systems or components, ensuring their long-term health.
+* Identify and proactively mitigate technical risks and challenges.
```

Comment:
not only technical, but I think they are responsible for also identifying these in the area of processes, team dynamics, working strategies, team morale, etc.